daemon/algod/server.go

```diff
@@ -82,7 +82,25 @@ func (s *Server) Initialize(cfg config.Local, phonebookAddresses []string, genes
 	lib.GenesisJSONText = genesisText
 
 	liveLog := filepath.Join(s.RootPath, "node.log")
+	// if the hotDataDir is explicitly set, use it for logging
+	if cfg.HotDataDir != "" {
+		liveLog = filepath.Join(cfg.HotDataDir, "node.log")
+	}
+	// if the LogFilePath is explicitly set, use it
+	if cfg.LogFilePath != "" {
+		liveLog = cfg.LogFilePath
+	}
```

Comment:
There is also a `-o` option that the docker container uses in order to write logs to stdout.

Reply:
I'm not sure why that `-o` would be affected by changing the log file path here, is there more to be done?